### PR TITLE
POL-190 - Update FSM policies with new endpoint

### PR DESCRIPTION
**Pull request by chandravurubindi**
Update FSM policies with Credential service and Endpoint

### Description

Update FSM policies with new endpoint and Credential Service

### Issues Resolved

Update FSM policies with new endpoint and Credential Service
Update readme to replace credential instructions with instructions to use credential service

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### Running policy Link
SaaS Manager - Suspicious Users:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d85b2e8dac600012ec324

SaaS Manager - User Status Change:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d87231ccef500019d6033

SaaS Manager - Unsanctioned Spend:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d865f1ccef500019d6032

SaaS Manager - Renewal Reminder:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e8d840ee8dac600012ec320

Note:Tested all these policies with Credential Service and end point change (hard code Org ID value with 28010, since we don't have access to One Flexera Demo environment and other Org IDs:1105 and 27730 are not working).

### CODE REVIEW

File: saas/fsm/renewal_reminder/README.md [hunk 1]
```diff
@@ -19,11 +19,21 @@ This policy has the following input parameters required when launching the polic
 - *Upcoming Number of Days* - If an application renewal is due in the upcoming time period, raise an incident
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify
 
-## Policy Actions
+## Prerequisites
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
Comment (cdwilhelm) [inline saas/fsm/renewal_reminder/README.md:25]: Looks like it was carried over from another policy

File: saas/fsm/suspicious_users/README.md [hunk 2]
```diff
@@ -18,11 +18,21 @@ This policy has the following input parameters required when launching the polic
 
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify
 
-## Policy Actions
+## Prerequisites
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
Comment (cdwilhelm) [inline saas/fsm/suspicious_users/README.md:24]: carried over from another policy

File: saas/fsm/unsanctioned_spend/README.md [hunk 3]
```diff
@@ -19,11 +19,21 @@ This policy has the following input parameters required when launching the polic
 - *Number of Days Back* - Any unsanctioned expenses discovered during this time period will raise an incident
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify
 
-## Policy Actions
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
Comment (cdwilhelm) [inline saas/fsm/unsanctioned_spend/README.md:25]: same as the others.  carried over from another policy

File: saas/fsm/user_status_change/README.md [hunk 4]
```diff
@@ -19,11 +19,22 @@ This policy has the following input parameters required when launching the polic
 - *Number of Days Back* - If a user's status changes to inactive during this time period, those user accounts will raise an incident
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify
 
-## Policy Actions
+## Prerequisites
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```
Comment (cdwilhelm) [inline saas/fsm/user_status_change/README.md:25]: same as the others

File: saas/fsm/suspicious_users/README.md (hunk 2 above)
Comment (chandravurubindi) [reply saas/fsm/suspicious_users/README.md]: updated now.

File: saas/fsm/renewal_reminder/README.md (hunk 1 above)
Comment (chandravurubindi) [reply saas/fsm/renewal_reminder/README.md]: Sorry,now its updated.

File: saas/fsm/unsanctioned_spend/README.md (hunk 3 above)
Comment (chandravurubindi) [reply saas/fsm/unsanctioned_spend/README.md]: updated

File: saas/fsm/user_status_change/README.md (hunk 4 above)
Comment (chandravurubindi) [reply saas/fsm/user_status_change/README.md]: corrected.

**Comment by chandravurubindi** [comment]
> Sorry, some more minor changes. I was reading the whole README file, not just your changes and saw that the Functional Details include obsolete details.
> 
> > Retrieve a Flexera IAM refresh token for authentication
> > The Flexera IAM refresh token must then be stored as a RightScale Credential in the account in which this policy will be applied. The credential must be named FSM_TOKEN.

@cdwilhelm - Thanks, I have removed the above details and added the below change in the Functional Details.
>>Please contact your Flexera Customer Success Manager for assistance to generate your FSM token.
Let me know for any other comments, Thanks.

**Pull request merged**